walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,92 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
 
 public final class FindMeetingQuery {
+
+  /**
+   * > 0 : false
+   * = 0 : true
+   */
+  private List<Integer> isAvailableAtMinute;
+
+  /**
+   * Worst Case
+   * Time Complexity: O(3 * 24 * 60 + events.size() * attendees.size())
+   *
+   * @param events Collection of Events
+   * @param request MeetingRequest
+   * @return Collection of potential Time Ranges for Meeting
+   */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+
+    // initialize array with true
+    isAvailableAtMinute = initializeIsAvailableAtMinuteArray(0);
```

Comment:
isMinuteAvailableForAllAttendees can be a better name?

Reply:
Sure. Shall update the same.

Reply:
Done.